websites/Z/ZDFmediathek/presence.ts

```diff
@@ -0,0 +1,131 @@
+          )
+          .innerHTML.replace(/ {2}/g, " ")
+          .replace(/ im Livestream/g, "")
+          .replace(/ Livestream/g, ""),
+        videoInfoResults = document.getElementsByClassName(
+          "zdfplayer-teaser-title"
+        );
+
+      let videoInfoTag = null;
+      for (let i = 0; i < videoInfoResults.length; i++) {
+        if (
+          videoInfoResults[i].innerHTML
+            .toLowerCase()
+            .includes(` ${mediathekLivechannel.toLowerCase()} `) ||
+          videoInfoResults[i].innerHTML
+            .toLowerCase()
+            .includes(`>${mediathekLivechannel.toLowerCase()}<`)
+        ) {
+          videoInfoTag = videoInfoResults[i].innerHTML;
+          break;
+        }
+      }
+      const videoTitle = videoInfoTag
+        .substring(videoInfoTag.lastIndexOf(">") + 1, videoInfoTag.length - 1)
+        .trim();
+
+      presenceData.largeImageKey = mediathekLivechannel.toLowerCase();
+      presenceData.smallImageKey = "live";
+      presenceData.smallImageText = "Live";
+      presenceData.details = `${mediathekLivechannel} Live`;
+      presenceData.state = videoTitle;
+      presenceData.startTimestamp = elapsed;
+      presenceData.buttons = [
+        { label: (await strings).buttonWatchStream, url: prevUrl }
+      ];
+
+      if (liveVideo.paused) {
+        presenceData.smallImageKey = "pause";
+        presenceData.smallImageText = (await strings).pause;
+        presenceData.startTimestamp = 0;
+        presenceData.endTimestamp = 0;
+      }
+    } else {
+      // Video-on-demand
+      presenceData.largeImageKey = "zdf";
+      presenceData.smallImageKey = "play";
+      presenceData.smallImageText = (await strings).play;
+
+      const videoInfoTag = document.querySelector(
+          ".zdfplayer-teaser-title"
+        ).innerHTML,
+        showTitleTag = videoInfoTag.substring(
+          videoInfoTag.indexOf(">") + 1,
+          videoInfoTag.lastIndexOf("<")
+        ),
+        episodeTitle = videoInfoTag
+          .substring(videoInfoTag.lastIndexOf(">") + 1, videoInfoTag.length - 1)
+          .trim(),
+        timestamps = presence.getTimestamps(
+          Math.floor(video.currentTime),
+          Math.floor(video.duration)
+        );
+
+      presenceData.state = episodeTitle;
+      presenceData.details = showTitleTag.includes("|")
+        ? showTitleTag.substring(
+            showTitleTag.indexOf("|") + 1,
+            showTitleTag.length
+          )
+        : showTitleTag;
+      [, presenceData.endTimestamp] = timestamps;
+      presenceData.buttons = [
+        { label: (await strings).buttonWatchVideo, url: prevUrl }
+      ];
+      if (video.paused) {
+        presenceData.smallImageKey = "pause";
+        presenceData.smallImageText = (await strings).pause;
+        presenceData.startTimestamp = 0;
+        presenceData.endTimestamp = 0;
```

Comment:
dont set to 0
do:
delete presenceData.startTimestamp

Reply:
Should be have taken care of by latest commit.
Thanks for the advice 👍 
I'm also gonna change this in my last presence contribution (ARD Mediathek #3950) via PR.